### PR TITLE
Minor doc fixes

### PR DESCRIPTION
**Pull request by benallard**
| | |
|:---|:---|
|**_Originating Project/Creator_**| me |
|**_Affected Component_**| doc |
|**_Affected Architectures(s)_**|  none |
|**_Related Issue(s)_**| none |
|**_Has Unit Tests (y/n)_**| n |
|**_Builds Without Errors (y/n)_**| y |
|**_Unit Tests Pass (y/n)_**|y  |
|**_Documentation Included (y/n)_**| y |

---
## Change Description

Typos, link fixes in doc.

## Rationale

Easier to read and follow links.

## Testing/Review Recommendations

Read doc, follow links.

## Future Work

Further reading of the docs and understanding of the stuff.

### CODE REVIEW

File: docs/Tutorials/GettingStarted/Tutorial.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ Most developers wish to run F´ on embedded hardware. This is done by generating
 toolchain by providing the toolchain argument. The above invocations assume the default "native" toolchain.
 
 Here, the "raspberrypi" toolchain supplied by F´ is used. This will generate a cross-compile for the raspberry pi
-architecture.  The user should ensure to follow the Raspberry PI setup steps found at: [RPI](https://github.com/nasa/fprime/blob/master/RPI/README.md)
+architecture.  The user should ensure to follow the Raspberry PI setup steps found at: [RPI](../../../RPI/README.md)
```
Comment (LeStarch) [inline docs/Tutorials/GettingStarted/Tutorial.md:121]: These docs are rendered as part of our documentation.  Links like this that leave the `docs` directory need to be a full url to the github path.

File: docs/Tutorials/GettingStarted/Tutorial.md [hunk 2]
```diff
@@ -195,7 +195,7 @@ This happens any time the "install" or "build" command are run on a deployment.
 
 The user can also build a component or deployment for a cross-compile by specifying the toolchain.  A previous generate
 for that toolchain should have been run. Again for deployments, the user typically should run "install", see below. These
-steps require the setup describe here: [RPI](https://github.com/nasa/fprime/blob/master/RPI/README.md)
+steps require the setup described here: [RPI](../../../RPI/README.md)
```
Comment (LeStarch) [inline docs/Tutorials/GettingStarted/Tutorial.md:198]: These docs are rendered as part of our documentation.  Links like this that leave the `docs` directory need to be a full url to the github path.

**Review by LeStarch — CHANGES_REQUESTED** [review]
Thanks for the edits!  Would you mind replacing the original full urls.  This will prevent broken links here: https://nasa.github.io/fprime/Tutorials/GettingStarted/Tutorial.html

File: docs/Tutorials/GettingStarted/Tutorial.md (hunk 1 above)
Comment (benallard) [reply docs/Tutorials/GettingStarted/Tutorial.md]: Sure. didn't knew about that.

**Review by LeStarch — APPROVED** [review]
Thanks for making those fixes!

**Pull request merged**